installer/common/requirements.sh

```diff
@@ -16,4 +16,8 @@
 
 # Perform checks of utilities required to perform the installation
 
+if [ "${BASH_VERSINFO}" -lt 4 ] | [ "${BASH_VERSINFO[0]}" -lt 4 ]; then
```

Comment:
LGTM. I assume the left part covers Linux and the right OS X.

Reply:
Here's the output of the check on MacOS after `brew install bash` to get the latest instead of the builtin version:

```
$ echo $BASH_VERSINFO
5
```

Reply:
@agemooij Does the install go through with bash 5 in your MacOS ?

Reply:
I get nothing when I do that (macOS):
```

Reply:
BASH_VERSINFO is not set for me @debasishg but I do have bash 5 installed (macOS), so I'm not sure if this will always work:
```
> bash --version
GNU bash, version 5.0.11(1)-release (x86_64-apple-darwin19.0.0)
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
> echo $BASH_VERSINFO

```

Reply:
@RayRoestenburg Can u pls check what u get with `echo $BASH_VERSION` - I can use that too but just have to do some additional shell stuff to extract the exact version.

Reply:
I have version 4 installed and I get ..

```
$ echo $BASH_VERSINFO
4
```

Reply:
@RayRoestenburg Can u pls check what u get with `echo ${BASH_VERSINFO[0]}` ?

Reply:
@RayRoestenburg you should have[ `${BASH_VERSINFO[0]}`](https://stackoverflow.com/questions/29060076/how-to-get-bash-version-number-in-os-x) set.

Reply:
@debasishg another more robust option I suspect is: 
```BASHFULL=$(bash --version | head -1 | awk '{print $4}')
${BASHFULL%%.*}```

Reply:
@skonto yeah .. I tried that too .. also this .. 

```
$ ver=$(echo $BASH_VERSION | cut -d'.' -f1)
$ echo $ver
4
```

Reply:
The script is run using `/usr/bin/env bash` so then the context s bash and you will get those env vars set correctly. They will not work if you execute them from another shell, like my default shell, which is zsh.